src/libs/getCurrentPosition/index.android.ts

```diff
@@ -0,0 +1,58 @@
+// https://github.com/Richou/react-native-android-location-enabler/issues/40
+// If we update our react native version, we need to test this file again
+import Geolocation from '@react-native-community/geolocation';
+import RNAndroidLocationEnabler from 'react-native-android-location-enabler';
+import {GeolocationErrorCode, GetCurrentPosition} from './getCurrentPosition.types';
+
+Geolocation.setRNConfiguration({
+    skipPermissionRequests: false,
+    authorizationLevel: 'whenInUse',
+    locationProvider: 'auto',
+});
+
+const getCurrentPosition: GetCurrentPosition = (success, error, config) => {
+    // Prompt's the user to enable geolocation permission with yes/no options
+    // If the user selects yes, then this module would enable the native system location
+    // Otherwise if user selects no, or we have an issue displaying the prompt, it will return an error
+    RNAndroidLocationEnabler.promptForEnableLocationIfNeeded({
+        interval: 2000, // 2 seconds. This represents the time interval after which our app expects an
+        // updated location. We don't depend on this here, but adding it here as its a required prop.
+        fastInterval: 1, // 1 millisecond. This represents the shortest amount of time our app is willing
+        // to wait for location update from the library. Passing 0 here won't work, as the library internally
+        // uses a ternary that would short if we pass 0, and it will then use its default value.
```

Comment:
Please move these long long comments on top of the code instead of doing this weird inline comment that goes beyond one line